documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+
+.Scan OVA files in the root folder of an NFS share
+
+Files held directly on the NFS share are scanned, not requiring a folder. link:https://issues.redhat.com/browse/MTV-680[(MTV-680)]
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+// {project-short} {project-z-version} is a maitenance release. There are no new features.
+This release has the following features and improvements:
+
+.Cross-Provider Features
+
+This is a collection of new features that provide functionality across different cloud providers. link:https://issues.redhat.com/browse/MTV-802[(MTV-802)]
+
+..Replace templates with flavors
+
+Kubevirt API introduces `VirtualMachineClusterFlavor` and `VirtualMachineFlavor` CDIs. Flavors are meant to replace Template, which is only available on OpenShift. link:https://issues.redhat.com/browse/MTV-206[MTV-206]
+
+..Integrate volume populators in remote migrations
+
+Integrate volume populator for `oVirt` and OpenStack, together with `virt-v2v` on `el9` into a CDI to migrate remote OCP clusters. link::https://issues.redhat.com/browse/MTV-721[MTV-721]
+
+..VM posted with `Running` state
+
+{project-short} shows each VM's migration `Running` state during the migration. link:https://issues.redhat.com/browse/MTV-794[MTV-794]
+
+..Review and configure migrated VMs
+
+The {project-short} web console shows the name and migration status of each VM. link:https://issues.redhat.com/browse/MTV-843[MTV-843]
+// 
+// ..CNV InstanceTypes
+// 
+// This looks like it repeats MTV-206 above
+
+.RH-SDL
+
+{project-short} conforms with the Red Hat Secure Development Lifecycle (RH-SDL) for users to follow an implementation plan that adopts security controls during the product lifecycle. link:https://issues.redhat.com/browse/MTV-801[(MTV-801)]
+// Do we have public links to these documents?
+// Additional RH-SDL resources:
+// * link:https://docs.google.com/document/d/1QMrM5ac2sbecmy7lYHA8S6p8L8ivVwHlgdcspy-Z4VE/edit#heading=h.66y4kqbj468a[Red Hat Secure Development Lifecycle Implementation Plan]
+// * link:https://docs.google.com/presentation/d/1CnIq-MHgEoq_1QgaFU5uoOfZ7ZOnNzxPk9OdDUe4Me8/edit#slide=id.g1a5a54f838a_0_1509[Red Hat Secure Development Lifecycle Introduction]
+// * Link:https://docs.google.com/presentation/d/19H3tSzZ1pSGGwhSoZn3CFgyLQcBWbePAK0_5J4NHUGw/edit#slide=id.g22dc74ad918_0_740[Red Hat Secure Development Lifecycle Planning and Schedule]
+// * link:https://docs.google.com/presentation/d/1DOxSd5hpwNntypX5DUd3JRmP8wIJL_RVxOJfDo0Nxck/edit#slide=id.g13028f60288_0_0[Secure Development - Introduction to SSML]
+// * link:https://gitlab.cee.redhat.com/users/auth/geo/sign_in[Closed link]
+// * link:https://docs.engineering.redhat.com/display/PRODSEC/Secure+Development+training[Secure Development Training]
```

Comment:
```suggestion
```

Reply:
This looks messy and might confuse the next writer. Keep your docs tidy

Reply:
Whole issues removed